app/helpers/phase_banner_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module PhaseBannerHelper
+  def phase_banner_tag_text(env = Rails.env)
+    return "Beta" if env == "production"
+
+    env
+  end
+
+  def phase_banner_tag_colour(env = Rails.env)
+    {
+      "development" => "red",
```

Comment:
could we include deployed development?

Reply:
maybe also review?

Reply:
oh review, good call. Was going to omit deployed_development as it'll be short-lived

Reply:
colour nominations for review?

Reply:
it's purple, for now!